e2e/e2e_test.go

```diff
@@ -95,8 +95,8 @@ func Test_http_body(t *testing.T) {
 	for _, tc := range []struct {
 		op, expBody string
 	}{
-		{op: "append", expBody: `[original body][this is appended body]`},
-		{op: "prepend", expBody: `[this is prepended body][original body]`},
+		{op: "append", expBody: `[original body][this is appended body][this is appended body]`},
```

Comment:
could we replace/append/prepend only at either Response or Request (let's control the behavior with other header), then there won't be no ambiguity in the response body which part of Plugin create what?

Reply:
like `curl -XPUT localhost:18000 --data '[initial body]' -H "buffer-operation: prepend" -H "buffer-replace-at: response"` to make replacement happen at response where `buffer-replace-at: response` controls the behavior.


Also, please make change to  https://github.com/tetratelabs/proxy-wasm-go-sdk/blob/main/examples/http_body/README.md

Reply:
Sure updated